inc/userinjection.class.php

```diff
@@ -185,6 +185,45 @@ public function processAfterInsertOrUpdate($values, $add = true, $rights = [])
             /** @phpstan-ignore-next-line */
             $DB->query($query); // phpcs:ignore
         }
+
+        if (isset($values['User']['profiles_id'])) {
+            $profile_user = new Profile_User();
+            $profile_users = $profile_user->find([
+                'users_id' => $values['User']['id'],
+                'entities_id' => $values['User']['entities_id'],
+            ]);
+            $is_new_profile_user = true;
+            foreach ($profile_users as $profile) {
+                if (
+                    in_array('users_id', $profile)
+                    && in_array('profiles_id', $profile)
+                    && in_array('entities_id', $profile)
+                ) {
+                    $is_new_profile_user = false;
+                    $profile_user->update([
+                        'id' => $profile['id'],
+                        'is_default_profile' => 1
+                    ]);
+                }
```

Comment:
This is always run (`$profile` alwys have `users_id`, `profiles_id`, `entities_id`)
so it's always updated `'is_default_profile' => 1`